packages/bot/src/stores/flyout-store.js

```diff
@@ -54,6 +62,13 @@ export default class FlyoutStore {
         this.flyout_content = observable(xml_list_group);
         this.setFlyoutWidth(processed_xml);
         this.setVisibility(true);
+
+        // apparently setFlyoutWidth doesn't calculate blocks dimentions until they're visible
```

Comment:
Can we add a `TODO: Find a proper solution` here?

Reply:
done